hdr_explainer.md

```diff
@@ -1,33 +1,35 @@
 # HDR Capability Detection
-HDR capabilities are detected with a combination of CSS and MediaCapabilities APIs.  Both APIs should be called to determine if HDR can be correctly presented. 
+
+HDR capabilities are detected with a combination of CSS and MediaCapabilities APIs.  Both APIs should be called to determine if HDR can be correctly presented.
 
 Please note that both of these APIs are fairly new and are not yet implemented by all user agents. See Implementations section below.
 
 # CSS: is the display HDR capable?
 
-Media Queries Level 5 defines a `dynamic-range` feature, with values `"standard"`  and `"high"`. 
+Media Queries Level 5 defines a `dynamic-range` feature, with values `"standard"`  and `"high"`.
 
 https://www.w3.org/TR/mediaqueries-5/#descdef-media-dynamic-range
 
 ```
 let isScreenHDR = window.matchMedia('(dynamic-range: high)').matches;
 ```
 
-This indicates whether the current display is HDR capable. 
+This indicates whether the current display is HDR capable.
 
 # CSS: video-* prefixed MediaQueries
+
 User agents on TVs tend to offer video capabilities at the native resolution and dynamic range (ex: 4k HDR) while offering much lower capabilities for the rest of the web page (maybe 1080p SDR). To surface these distinct capabilities, Media Queries Level 5 defines features prefixed by "video-".
 
 https://www.w3.org/TR/mediaqueries-5/#video-prefixed-features
 
-This includes `video-dynamic-range`. On televisions, this value may be `"high"` while the non-prefixed `dynamic-range` remains `"standard"`. 
+This includes `video-dynamic-range`. On televisions, this value may be `"high"` while the non-prefixed `dynamic-range` remains `"standard"`.
 
-On desktop and mobile devices, values for video-* features will generally match their non-prefixed counterparts. 
-
-The sizing video-\* prefixed features are still under discussion, including `video-width`, `video-height`, and `video-resolution`. See https://github.com/w3c/csswg-drafts/issues/5044
+On desktop and mobile devices, values for video-* features will generally match their non-prefixed counterparts.
 
+The sizing video-* prefixed features are still under discussion, see CSS issues [#4471](https://github.com/w3c/csswg-drafts/issues/4471), [#4678](https://github.com/w3c/csswg-drafts/pull/4678), [#5044](https://github.com/w3c/csswg-drafts/issues/5044), and [#6891](https://github.com/w3c/csswg-drafts/issues/6891). Rather than add `video-width`, `video-height`, and `video-resolution` media queries, the current proposal is to expose a `deviceVideoPixelRatio` property.
```

Comment:
```suggestion
The sizing video-* prefixed features are still under discussion, see CSS issues [#4678](https://github.com/w3c/csswg-drafts/pull/4678), [#5044](https://github.com/w3c/csswg-drafts/issues/5044), and [#6891](https://github.com/w3c/csswg-drafts/issues/6891). Rather than add `video-width`, `video-height`, and `video-resolution` media queries, the current proposal is to expose a `deviceVideoPixelRatio` property.
```